cfme-rhconsulting-scripts.spec

```diff
@@ -1,6 +1,6 @@
 Name:      cfme-rhconsulting-scripts
-Version:   0.8
-Release:   2
+Version:   0.9
```

Comment:
The /var/www/miq/vmdb/lib/tasks/rhconsulting_illegal_chars.rb and /var/www/miq/vmdb/lib/tasks/rhconsulting_options.rb files need to be added to the %files section so they get installed.

Also, the %changelog section should be updated.